docs/source/model_design/risk_factors.rst

```diff
@@ -48,8 +48,111 @@ Definitions
 Theoretical Minimum Risk Exposure Level/Distribution (TMREL/D)
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-Relative Risk (RR)
+Measures of occurence
++++++++++++++++++++++
+
+This is a recap: Epidemiology is the study of the distribution and determinants of disease frequency in human populations. Simply put, it is the study of the *occurence* of illness. Measures of disease frequency are tools to describe how common an illness is (or outcome of an event) with reference to the size of the population at risk. They are used to count cases, in relation to a population and a measure of time. Outcomes can be infection, disease, disability, death, recovery or usage of health care. 
```

Comment:
It would be nice to have a link here to the section that this is a recap of. and maybe a todo to come back and decide how much redundancy we want after this page is no longer just an outline everyone is working on? is that over-ambitious to plan on trying to thin it out?

Reply:
ya, good idea